app/server/lib/Assistance.ts

```diff
@@ -485,11 +484,13 @@ class EchoAssistant implements Assistant {
  * Instantiate an assistant, based on environment variables.
  */
 export function getAssistant() {
-  if (process.env.OPENAI_API_KEY === 'test') {
+  const apiKey = process.env.ASSISTANT_API_KEY || process.env.OPENAI_API_KEY;
+
+  if (apiKey === 'test') {
     return new EchoAssistant();
   }
   if (process.env.OPENAI_API_KEY || process.env.ASSISTANT_CHAT_COMPLETION_ENDPOINT) {
-    return new OpenAIAssistant();
+    return new OpenAIAssistant(apiKey);
```

Comment:
Can you double check line 492? It doesn't fill that it is treating those two keys as the same..